cardano-config/src/Cardano/TracingOrphanInstances/Byron.hs

```diff
@@ -38,6 +39,28 @@ import           Ouroboros.Consensus.Byron.Ledger (TxId(..))
 --
 -- NOTE: this list is sorted by the unqualified name of the outermost type.
 
+instance ToObject ApplyMempoolPayloadErr where
```

Comment:
This can be done in a follow up PR, but we should write the `ToObject` instances for the subtypes of `ApplyMempoolPayloadErr`